File: C1908GLeThanhNghi/AP/25-11-2020/myApp/myApp/Models/SavingsAccount.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace myApp.Models
{
    public class SavingsAccount
    {
        public static double annualInterestRate;
        private float savingsBalance;
        public float SavingsBalance { get {
                return savingsBalance;
            } 
            set {
                savingsBalance = value;
            } 
        }
        public double CalculateMonthlyInterest() =>
            annualInterestRate * savingsBalance / 12.0;
        public static void ModifyInterestRate(double newValue) {
            annualInterestRate = newValue;
        }

    }
}

File: C2108G1/ADF1/20-09-2022/javade02/src/main/java/org/aptech/javade02/SavingsAccount.java
package org.aptech.javade02;

public class SavingsAccount {
    private Double balance;

    public Double getBalance() {
        return balance;
    }

    public SavingsAccount(Double balance) {
        this.balance = balance;
    }
    private static Float ANNUAL_INTEREST_RATE = 0.15f;
    public Double calculateMonthlyInterest() {
        double interestRate = balance * ANNUAL_INTEREST_RATE / 12.0;
        balance += interestRate;//danger !
        return interestRate;
    }
    public static void modifyInterestRate(Float newValue) {
        ANNUAL_INTEREST_RATE = newValue;
    }
}
